### PR TITLE
better mobile support and deck in grid view

### PR DESCRIPTION
**Pull request by rotwurstesser**
changes: css formatting, css grid implementation (might need to add fallbacks since we dont work with compiled css in this project...) touch support

open: 
-perhaps add fallbacks and browser vendor prefixes (-ms,-o) for the new CSS stuff

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,13 +1,69 @@
-.scryfall_hover-zoom { transition: transform .2s; width: 200px; position: relative; z-index: 10; }
-.scryfall_hover-zoom:hover { transform: scale(1.5); z-index: 100; }
-.scryfall_hover-zoom--container { position: relative; }
```
Comment (dbu) [inline styles.css]: i would prefer to not reformat these to keep the diff focussed on the new features.

Comment (rotwurstesser) [reply styles.css]: i will change it back then ^^ probably won't get to work on it before 🇷🇺 though... but i will need to find out how to implement my hook first... is there a way i can work directly on a live demo ? (so i can play around with the php as well)

File: utilitiesformtg.js [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+        document.getElementById(id).src=back;
     } else {
         document.getElementById(id).src=front;
     }
-};
\ No newline at end of file
+};
+
+//bind touch event
+//TODO: check how we do this with dynamic content
```
Comment (dbu) [inline utilitiesformtg.js]: is this still open?

File: utilitiesformtg.js [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
+setInterval(function  () {
```
Comment (dbu) [inline utilitiesformtg.js:15]: can you please add a doc comment explaining why this needs to be done in an interval rather than just once?

Comment (rotwurstesser) [reply utilitiesformtg.js:15]: like this? If there would be an option to change the event and add it inline to the generated tags, would you have time to look at that with me?

Comment (dbu) [reply utilitiesformtg.js:15]: the html is defined here: https://github.com/dbu/wordpress-mtg/blob/master/utilitiesformtg.php#L242 if we can add some element that helps, sure!

is the interval thing "polling"? do we stop polling once the view is loaded?

Comment (rotwurstesser) [reply utilitiesformtg.js:15]: we would need a "touchstart" event on the cards or one of the elements around them (not sure which) like this: https://www.w3schools.com/jsref/event_touchstart.asp
and then we would need a script tage somewhere on the page which binds the function into a global space for example: window.__scryFallPreviewTouchEvent(event){
}
then we would have our function binding on all cards and need no interval anymore at all, we just need to be sure that the function is always loaded and at best its only loaded one time per page and always before we display the cards....

**Pull request merged**